app/App/Account/Account/Requests/TransactionRequest/TxAction/index.js

```diff
@@ -70,7 +69,7 @@ class TxSending extends React.Component {
                 <ClusterValue>
                   <span className='_txMainTransferringEq'>{'≈'}</span>
                   <span className='_txMainTransferringEqSymbol'>{'$'}</span>
-                  <span className='_txMainTransferringEqAmount'>{(displayValue * rateUSD).toFixed(2)}</span>
+                  <span className='_txMainTransferringEqAmount'>{!rateUSD ? '0.00' : (displayValue * rateUSD).toFixed(2)}</span>
```

Comment:
Would `?` be a more accurate default in this case?

Reply:
Might be, but then we should change all testnet values everywhere to `$?` - the send dapp currently displays `$0`

Reply:
Updated